src/answers-headless.ts

```diff
@@ -270,14 +277,46 @@ export default class AnswersHeadless {
     return results;
   }
 
-  executeFilterSearch(sectioned: boolean, fields: SearchParameterField[]): Promise<FilterSearchResponse> {
-    return this.core.filterSearch({
-      input: this.state.query.query || '',
+  async executeFilterSearch(
+    filterSearchId: string,
+    sectioned: boolean,
+    fields: SearchParameterField[]
+  ): Promise<void> {
```

Comment:
Can we continue to return a `Promise<FilterSearchResponse>`?

Reply:
I suggested we remove it since we were removing the response models, however after discussing this further we decided to leave this functionality in

Reply:
all good, I will add it back in!